src/olympia/reviewers/views.py

```diff
@@ -1121,6 +1130,46 @@ def unlisted_list(request):
                   unlisted=True, SearchForm=AllAddonSearchForm)
 
 
+def policy_viewer(request, addon, eula_or_privacy, page_title, long_title):
```

Comment:
Not convinced "viewer" is the right term here. maybe `verify_and_render_policy` or something like that?

Reply:
mmm.  I'm open to new names but it doesn't verify the policies (other than checking they exist) so imo that would be misleading.  
@eviljeff steps away from function name bikesheddin and merges 😁